examples/count_person/main.py

```diff
@@ -0,0 +1,74 @@
+"""This is a demo VQPy implementation listing and storing all red moving
```

Comment:
Should this comment be updated?

Reply:
This comment has been removed in this PR. And an introduction will be included in example Readme instead.